samples/04_gis_analysts_data_scientists/translating_story_map_from_one_language_to_another.ipynb

```diff
@@ -0,0 +1,4030 @@
+{
```

Comment:
<blockquote>The <code><strong>replace</strong></code> function is a recursive<a href="https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/files/#References" target="_blank">[2]</a> fun....</blockquote><p>space before [2]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Made the necessary changes</p>